### ISSUE TITLE
Not working with Riotjs .tag

### ISSUE THREAD

**Issue by mtt87** [AtomLinter/linter-eslint#718]
Not sure if the problem is related to to `language-riot-tag` https://github.com/dekimasoon/language-riot-tag
or if it's `linter-eslint`.

https://github.com/riot/riot/issues/1996

Using Atom + Linter + Linter-eslint I can't get it to lint my `.tag`
Tried with no success to rename the tags `.html` which I don't recommend anyway because you lose the `HTML (Riot tag)` automatic syntax which is useful (for example for atom-beautify)

My setup:
```
Atom 1.10.2
Linter 1.11.6
Linter-eslint 7.3.2
Language-riot-tag 0.2.2
```

I put all the components in `app/components/*.tag` and I'm using a `gulp-riot` task to compile them inside `scripts/*.js`
Everything works fine but can't get the linter to work.

My feeling is that the `linter-eslint` linter isn't being triggered for the `HTML (Riot tag)` syntax and also for the HTML syntax.

I think this is probably the part of code involved:
https://github.com/AtomLinter/linter-eslint/blob/master/lib/main.js#L28-L44
And then this:
https://github.com/dekimasoon/language-riot-tag/blob/master/grammars/riot-tag.cson

**Where do you think it's the problem?**

**Comment by Arcanemagus** [comment]
There are two parts to this, both of which seem simple enough to solve.

The reason `linter-eslint` isn't running on your file, is because the scope the file lists itself as (`text.html.riot`) isn't listed as the scopes it knows how to handle. You can tell it to run on these files by adding that to the `Scopes` option.

The other half of this issue is that ESLint itself doesn't know how to handle these files by default. For standard HTML files there is an ESLint plugin that extracts the `<source>` tag contents, sends it to ESLint and maps the results back to the original source ([`eslint-plugin-html`](https://www.npmjs.com/package/eslint-plugin-html). It seems this plugin claims support for `.tag` files, so you should just be able to add that plugin to your ESLint configuration to tell it how to handle these files.
